framework/src/main/java/io/github/kgress/scaffold/BaseWebElement.java

```diff
@@ -143,8 +147,7 @@
    * @param cssSelector the string value of the {@link By#cssSelector(String)}
    */
   public BaseWebElement(String cssSelector) {
-    this.setBy(By.cssSelector(cssSelector));
```

Comment:
You can restore these constructors to how they were before :)

Reply:
Is there something wrong with DRYing them up a little?